resources/views/searches/resources/groups/index.blade.php

```diff
@@ -1,5 +1,186 @@
-<!-- Content - OPEN -->
-<div class="container margin-top padding-bottom">
-    <img src="/img/mockup_groups.png" style="width: 100%" />
-</div>
-<!-- Content - CLOSE -->
+{{ Form::hidden('group_count', $search->groups->count()) }}
+<!-- If NO nogroup - OPEN --> 
+    <div id="no_group"class="card text-center">
+        <div class="card-body">
+
+            <img src="/img/add_search.png" width="300">
+
+            <h4 class="card-title margin-bottom text-secondary">
+                {{ __('messages.no_groups') }}
+            </h4>
+
+            <a href="#addGroupModal" class="btn btn-primary" role="button" data-toggle="modal"
+                <?php if (Auth::user()->profile == 'guest') { ?> style="display:none" <?php } ?> >
+                {{ __('actions.add') . ' ' . __('group.group') }}
+            </a>            
+        </div>
+    </div>
+<!-- If NO group - CLOSE -->
+
+<!-- If exists searches - OPEN -->
```

Comment:
Should be  <!-- If exists groups - OPEN -->

Reply:
ok